### ISSUE TITLE
Change data source: the number of cases, JHU to COVID-19 Data Hub

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#58]
We are using JHU dataset with `cs.DataLoader.jhu()` and `cs.JHUData()` now.
However, this dataset has critical errors (e.g. Italy: Confirmed=241184 and Recovered=11811 on 03Jul2020, Recovered << Confirmed) and the errors may not be corrected. So, we need change the source data that is maintained.

I found [COVID-19 Data Hub](https://covid19datahub.io/index.html) and this has [Python Interface](https://covid19datahub.io/articles/api/python.html).
We can retrieve the datasets as follows.
```
pip install covid19dh
```
```Python
import covid19dh
# Country level
country_df = covid19dh.covid19(country=None, level=1, verbose=False)
# For some countries, province-level data is included
province_df = covid19dh.covid19(country=None, level=2, verbose=False)
# List of citation
covid19dh.cite(country_df)
```
OxCGRT data and population values are included in this dataset.

In the next version, I will try to change the data source.
Thank you.

**Comment by lisphilar** [comment]
I notified COVID-19 Data Hub team in https://github.com/covid19datahub/COVID19/issues/87

**Comment by lisphilar** [comment]
This will be included in v2.4.0
